code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -90,14 +90,11 @@
 			new /obj/item/storage/toolbox/syndicate(src)
 
 		if("sabotage")
```

Comment:
is the sabotage bag suppose to be in here? You removed 2 C4 otherwise.

Reply:
Messing up the easiest things. smh